### PR TITLE
Fix client onboarding token usage explanation

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Bz: https://bugzilla.redhat.com/show_bug.cgi?id=2266845

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
image

**Comment by Chandan141996** [comment]
The current implementation doesn't align with the design recommendation. Please adjust it to match the proposed design. @TimothyAsirJeyasing 

**Recommendation**
Onboarding token

**Comment by TimothyAsirJeyasing** [comment]
image

File: locales/en/plugin__odf-console.json [hunk 1]
```diff
@@ -1110,8 +1110,8 @@
   "Client onboarding token": "Client onboarding token",
   "Copy to clipboard": "Copy to clipboard",
   "How to use this token?": "How to use this token?",
-  "To onboard the client cluster, the provider cluster requires the onboarding token.": "To onboard the client cluster, the provider cluster requires the onboarding token.",
-  "An onboarding token is needed to connect an additional OpenShift cluster to a Data Foundation deployment.": "An onboarding token is needed to connect an additional OpenShift cluster to a Data Foundation deployment.",
+  "An onboarding token is needed to connect an additional OpenShift cluster to an OpenShift Data Foundation deployment. Copy the generated token and use it for deploying OpenShift Data Foundation client operator on your OpenShift cluster.": "An onboarding token is needed to connect an additional OpenShift cluster to an OpenShift Data Foundation deployment. Copy the generated token and use it for deploying OpenShift Data Foundation client operator on your OpenShift cluster.",
```
Comment (SanjalKatiyar) [inline locales/en/plugin__odf-console.json:1113]: cc @Chandan141996 (no "Openshift Data Foundation", only "Data Foundation")...
```suggestion
  "An onboarding token is needed to connect an additional OpenShift cluster to an Data Foundation deployment. Copy the generated token and use it for deploying Data Foundation client operator on your OpenShift cluster.": "An onboarding token is needed to connect an additional OpenShift cluster to an OpenShift Data Foundation deployment. Copy the generated token and use it for deploying OpenShift Data Foundation client operator on your OpenShift cluster.",
```

Comment (TimothyAsirJeyasing) [reply locales/en/plugin__odf-console.json]: Done

**Review by GowthamShanmugam — COMMENTED** [review]
LGTM

**Review by bipuladh — APPROVED** [review]
/lgtm

**Comment by amr1ta** [comment]
As per latest design doc https://www.figma.com/design/78q7cZXg1Ob5IaK6E4pz9W/UX-Bugs---4.16?node-id=39-1571, the displayed explanation about how to use client onboarding token should be,


"""
An onboarding token is needed to connect an additional OpenShift cluster to an Openshift Data Foundation deployment. Copy the generated token and use it for deploying Openshift Data Foundation client operator on your Openshift cluster.


The token is valid for 48 hours and can only be used once.
"""


In web console its displaying as,


"""
An onboarding token is needed to connect an additional OpenShift cluster to a Data Foundation deployment. Copy the generated token and use it for deploying Data Foundation client operator on your Openshift cluster.


The token is valid for 48 hours and can only be used once.
"""

Instead of 'Openshift Data Foundation' currently 'Data Foundation' is displaying.

Please confirm if there is any further change in the design doc or the current message needs to be updated.

Validated with:
OCP: 4.16.0-ec.6
ODF: 4.16.0-108.stable

**Pull request merged**